table/src/main/java/com/bazaarvoice/emodb/table/db/astyanax/AstyanaxTableDAO.java

```diff
@@ -1394,9 +1470,82 @@ public void clearStashTokenRangeSnapshot(String stashId) {
         _stashTableDao.clearTokenRanges(stashId);
     }
 
+    @Timed (name = "bv.emodb.table.AstyanaxTableDAO.listMetadataChangedTables", absolute = true)
+    @Override
+    public Iterator<ChangedTable> listMetadataChangedTables(DateTime fromInclusive, DateTime toExclusive, LimitCounter limit) {
+        checkArgument(limit.remaining() > 0, "Limit must be > 0");
+        checkArgument(fromInclusive != null, "from date cannot be null");
+        checkArgument(toExclusive != null, "to date cannot be null");
+
+        LocalDate fromDate = LocalDate.of(fromInclusive.withZone(DateTimeZone.UTC).getYear(), fromInclusive.withZone(DateTimeZone.UTC).getMonthOfYear(),
+                fromInclusive.withZone(DateTimeZone.UTC).getDayOfMonth());
+        LocalDate toDate = LocalDate.of(toExclusive.withZone(DateTimeZone.UTC).getYear(), toExclusive.withZone(DateTimeZone.UTC).getMonthOfYear(),
```

Comment:
Why the complex conversion to `LocalDate`?  Everything can be expressed using `DateTime`.  Or, if there's a reason to convert, why not just `ZonedDateTime.ofInstant(Instant.ofEpochMilli(fromInclusive.getMillis()), ZoneOffset.UTC)`?